reliability/core-it/src/androidTest/kotlin/com/datadog/android/core/integration/tests/FeatureScopeTest.kt

```diff
@@ -156,6 +156,49 @@ class FeatureScopeTest : MockServerTest() {
         }.doWait(LONG_WAIT_MS)
     }
 
+    @Test
+    fun mustReceiveTheEvents_whenFeatureWrite_trackingConsentPendingToGranted_switched_asynchronously() {
```

Comment:
minor: I notice that the tests in this class do not follow our standard of M W?

Reply:
These are integration tests which ran with Junit4 and therefor we can't use that format in the function description.